### ISSUE TITLE
Profile header upload mirror buttons incorrect

### ISSUE THREAD

**Issue by LonMcGregor** [NodeBB/NodeBB#7095]
(I don't have the version/config info, as I don't administrate the forum I found this issue on)

- **Exact steps to cause this issue:**
1. Open your (logged in) user profile page
2. Upload a new header image
3. When presented with the "crop picture" window, click the flip buttons and watch what happens to the image
- **What you expected:**
The ↔ arrow flips the image horizontally (left becomes right), and ↕ does so vertically (top becomes down)
- **What happened instead:**
The arrows flip the image along the wrong axis (↔does up-down, and ↕ does left-right)

**Comment by barisusakli** [comment]
I can't reproduce this with an test image, could it be related to the image you are using? Can you share the image?

**Comment by LonMcGregor** [comment]
@barisusakli  I just tried it just now with a PNG and the bug doesn't occur.

I wonder if perhaps this is related to a JPEG orientation tag or something.

Here's a link to an image that exhibits the issue: https://1drv.ms/u/s!Ai5fATgM7eNphf1MLhA5mRcDRYZTOg
Click "View original" and download and use that one.

I can't upload it to somewhere easier like imgur or download from thumbnails as I think those will strip out the tag data that might be causing the problem.

**Comment by barisusakli** [comment]
Yeah the image has an exif rotation which doesn't play nice with cropperjs, even though checkOrientation is set to true. More info https://github.com/fengyuanchen/cropperjs#checkorientation

**Comment by barisusakli** [comment]
Lots of discussion and some possible solutions here https://github.com/fengyuanchen/cropper/issues/120

**Comment by mySebbe** [comment]
The original EXIF fixture is no longer downloadable, and the cropper stack has changed substantially: `master` is on CropperJS 1.6.2, `develop` on 1.6.3, with the blocked v2 upgrade in #13221. Server-side Sharp auto-orientation does not reproduce the browser-side flip controls. Could someone attach a fresh original JPEG and confirm the exact result on current `develop`? A fix should wait for that fixture and should be tested again after the CropperJS upgrade to avoid solving obsolete behavior.
